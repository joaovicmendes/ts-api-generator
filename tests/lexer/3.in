// Deve reportar símbolo não identificado
model [
    Student {
        name: string
        email: string
        ra: number
    }
    ExampleClass {
        student: Student,
        grade: number
    }
]
endpoints [
    Student {
        GET '/students'
        GET '/students/:ra'
        POST '/students'
        PUT '/students/:ra'
        DELETE '/students/:ra'
    }
    ExampleClass {
        GET 'class/:grade'
    }
]
