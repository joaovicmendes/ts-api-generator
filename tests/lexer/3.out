Linha 8: , - simbolo nao identificado
Compilação interrompida.
